S15/tests/dbbench/env.sh

```diff
@@ -1,2 +1,2 @@
+export GOPATH=$HOME/go
```

Comment:
Do you really need `go` for these tests?

Reply:
No

Reply:
But I leave it for now because I use `go env GOPATH` below...

Reply:
Not sure I understand the logic. You only use `go` to set `$PATH`?